clients/venice-admin-tool/src/main/java/com/linkedin/venice/datarecovery/StoreRepushCommand.java

```diff
@@ -0,0 +1,366 @@
+package com.linkedin.venice.datarecovery;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+
+/**
+ * StoreRepushCommand contains the details of executing/processing repush command.
+ */
+
+public class StoreRepushCommand {
+  private static final Logger LOGGER = LogManager.getLogger(StoreRepushCommand.class);
+  private static final String REPUSH_MODULE = "repush";
+  private static final String REPUSH_SOURCE_KAFKA = "kafka";
+
+  // Store name.
+  private String store;
+  private Params params;
+  private Result result;
+  private List<String> expectCmd;
+
+  public StoreRepushCommand() {
+  }
+
+  public StoreRepushCommand(String store, Params params) {
+    this.store = store;
+    this.params = params;
+    this.expectCmd = generateExpectCmd();
+  }
+
+  public void setParams(Params params) {
+    this.params = params;
+  }
+
+  public String getStore() {
+    return store;
+  }
+
+  public Result getResult() {
+    return result;
+  }
+
+  private List<String> generateRepushCommand() {
+    List<String> cmd = new ArrayList<>();
+    cmd.add(this.params.command);
+    cmd.add(REPUSH_MODULE);
+    cmd.add(REPUSH_SOURCE_KAFKA);
+
+    if (store != null) {
+      cmd.add("--store " + store);
+    }
+
+    if (params.fabric != null) {
+      cmd.add("--fabric " + params.fabric);
+    }
+
+    if (params.rewindTimeOverrideSeconds != -1) {
+      cmd.add("--rewind_time_override_seconds " + params.rewindTimeOverrideSeconds);
+    }
+
+    if (params.fabricGroup != null) {
+      cmd.add("--fabric_group " + params.fabricGroup);
+    }
+
+    if (params.force) {
+      cmd.add("--force");
+    }
```

Comment:
I'm not sure if this option should be exposed in Venice codebase. It seems leaky to me

Reply:
We can fix this by introducing EXTRA_COMMAND_ARGS as you mentioned.